### PR TITLE
Update for the project name and skill name

### PR DESCRIPTION
**Pull request by chi666666**
accepted

### CODE REVIEW

**Comment by chi666666** [comment]
accepted

File: index.md [hunk 1]
```diff
@@ -2,10 +2,10 @@
 
 Collaborative software engineer
 
-## My projects
+## Chi's projects
 
 * Collaborative Software Project in Computational Physicis
 
-## My skills
+## Chi's skills
```
Comment (chi666666) [inline index.md:9]: sttyty

**Review by chi666666 — COMMENTED** [review]
Thank you for these great suggestions for my portfolio website! I have a few reservations about some proposed changes. Please take a look and let me know what you think.

**Comment by chi666666** [comment]
dsffs

**Review by chixi99 — CHANGES_REQUESTED** [review]
sdasfsa

**Comment by chixi99** [comment]
fsdfsdf

**Pull request merged**